instrumentation/apolloconfig-apolloclient-1.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.ctrip.framework.apollo")
+    module.set("apollo-client")
+    versions.set("[1.0.0,2.3.0]")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  compileOnly("com.google.auto.value:auto-value-annotations")
+  annotationProcessor("com.google.auto.value:auto-value")
+
+  library("com.ctrip.framework.apollo:apollo-client:1.1.0")
+
+  testImplementation(project(":testing-common"))
+
+  latestDepTestLibrary("com.ctrip.framework.apollo:apollo-client:1.1.+")
```

Comment:
We run tests with the version used in `library` and the latest release. Here you limit the latest release to `1.1.2`. Any reason you don't want to to test with `2.3.0`. Tests seem to pass with `2.3.0`, only need to add `.allowClass("com.google.inject.internal.aop.ChildClassDefiner$ChildLoader")` to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/3fd27aa4d134d0f39bba5166eb94dc1b64bd9a1d/javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/ignore/AdditionalLibraryIgnoredTypesConfigurer.java#L256
Since you have a weird situation where instrumentation works with `1.0` but tests require `1.1` to compile (but not run). You could use
```
library("com.ctrip.framework.apollo:apollo-client:1.0.0")
// tests require 1.1.0 to compile but run with 1.0.0
testCompileOnly("com.ctrip.framework.apollo:apollo-client:1.1.0")
```
also remove `latestDepTestLibrary`